trento/migration/systemd-install.md

```diff
@@ -408,6 +404,12 @@ This is a basic guide for creating a self-signed certificate for use with Trento
    ```bash
    systemctl enable --now nginx
    ```
+   
+1.  Copy the generated trento.crt in a location accessible by nginx:
```

Comment:
Any reason why you want to put it here ? 
I would suggest to move it in line 357 like it was before :) 
just to keep the key and cert steps together

Reply:
Because the new suggested location /etc/nginx/ssl/certs only exists once nginx has been installed, right?

Reply:
Because the new location for the crt file only exists once nginx has been installed, right?

Reply:
Fair point, it is better to install nginx and then handle the certificates and keys.... 
Actually would it not make sense to reorder the structure of this chapter?

Now we have: 
- Prepare SSL certificate for NGINX
- Install and configure NGINX

to 
- Install and configure NGINX
- Prepare SSL certificate for NGINX

The only thing what the user must do after cp is to: 
Reload Nginx to apply changes:
```
systemctl reload nginx
```

All the other steps basically stay the same, but this is a bit of effort tbh, we are good to go with what you have now, we also could restructure it in the future :)
So up to you :)

Reply:
I also think that reordering the structure of the chapter is the best way to go.